commands/operator-sdk/cmd/cmdutil/util.go

```diff
@@ -29,8 +29,17 @@ import (
 	yaml "gopkg.in/yaml.v2"
 )
 
-const configYaml = "./config/config.yaml"
-const tmpDockerfile = "./tmp/build/Dockerfile"
+type OperatorType int
+
+const (
+	configYaml    = "./config/config.yaml"
+	gopkgToml     = "./Gopkg.toml"
+	tmpDockerfile = "./tmp/build/Dockerfile"
+)
+const (
+	goOperatorType OperatorType = iota
```

Comment:
Rather than redefining these enums in `up/local.go`, it probably makes more sense to export them from here (where `OperatorType` is defined). I'd also suggest a different naming convention to make them sort together in IDEs, godoc, etc.

```go
const (
  OperatorTypeGo OperatorType = iota
  OperatorTypeAnsible
)
```

Then in `up/local.go`, you can remove the const declarations and use these directly like `cmdutil.OperatorTypeGo`